src/main/java/org/broadinstitute/hellbender/dev/utils/ReadRecord.java

```diff
@@ -0,0 +1,274 @@
+package org.broadinstitute.hellbender.dev.utils;
+
+import com.google.api.services.genomics.model.CigarUnit;
+import com.google.api.services.genomics.model.LinearAlignment;
+import com.google.api.services.genomics.model.Position;
+import com.google.api.services.genomics.model.Read;
+import com.google.common.collect.Iterables;
+
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SAMTag;
+import htsjdk.samtools.SAMTagUtil;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Read -> SAMRecord.
```

Comment:
Comment needs to be more descriptive. Where is this ReadRecord being used, why does it exist?  When would someone use it?